.flake8

```diff
@@ -3,4 +3,4 @@
 # W504 is the "line break before binary operator" error and is disabled since we use
 # the black code formatter.
 # W503 is disabled by default, but must be disabled explicitly when using `ignore`.
-ignore = E501, W503, W504
+ignore = E501, W503, W504, F401
```

Comment:
Disables an error about module imported but not used that is hard to fix without changing the templates.

The python SDK is due for an overhaul soon, so not worth digging in.